lib/_emerge/depgraph.py

```diff
@@ -3777,11 +3787,45 @@ def _eliminate_rebuilds(self):
                 modified = True
                 parent_atoms = []
                 for parent, parent_atom in self._dynamic_config._parent_atoms[pkg]:
-                    priorities = self._dynamic_config.digraph.nodes[pkg][1][parent][:]
+                    try:
+                        priorities = self._dynamic_config.digraph.nodes[pkg][1][parent][
+                            :
+                        ]
+                    except KeyError:
+                        optional_msg = " ({} previously removed from graph)"
+                        warnings.warn(
+                            f"_eliminate_rebuilds inconsistency: parent priorities missing for {parent} -> {pkg} edge"(
```

Comment:
When running tests for https://github.com/gentoo/portage/pull/1234 (`lib/portage/tests/update/test_move_resolve.py`), I noticed:
```
lib/_emerge/depgraph.py:3797
  /home/sam/git/portage/lib/_emerge/depgraph.py:3797: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?
    f"_eliminate_rebuilds inconsistency: parent priorities missing for {parent} -> {pkg} edge"(
```